deps/rabbitmq_management/priv/www/js/main.js

```diff
@@ -7,6 +7,8 @@ $(document).ready(function() {
      renderWarningMessageInLoginStatus(fmt_escape_html(error));
    } else {
       if (oauth.enabled) {
+        store_pref("oauth-return-to", url.hash);
```

Comment:
Maybe we can move `store_pref("oauth-return-to", url.hash);` inside `startWithOAuthLogin()`?

Reply:
Done, thanks.